### PR TITLE
fix section sort

### PR DESCRIPTION
**Pull request by nicolasfranck**
What it tries to fix: if you, in the org-admin, try to move the section headers, you'll get a javascript in the console.

Fix:

* `$.rails.ajax` should use option `type`, not `method`
* controller `OrgAdmin::PhasesController#sort` expects `phase[sort_order]`, not just `sort_order`
* concern incorrectly converted array of ids into array of arrays by prepending ids with a star.

### CODE REVIEW

File: app/javascript/src/orgAdmin/phases/show.js [hunk 1]
```diff
@@ -52,8 +52,8 @@ $(() => {
       // Post the section IDs to the server in their new order on the page.
       $.rails.ajax({
         url: $(this).data('url'),
-        method: 'post',
-        data: { sort_order: sectionIds },
+        type: 'post',
+        $.param({ 'phase[sort_order]': sectionIds }),
```
Comment (briri) [inline app/javascript/src/orgAdmin/phases/show.js:56]: I think you're missing `data: ` here. I receive a JS error with the above

**Review by briri — CHANGES_REQUESTED** [review]
looks good @nicolasfranck. Just make that one small change and we will be good 👍

**Comment by nicolasfranck** [comment]
Fixed.

Btw, how did you manage to merge in your development branch into my branch?

See https://github.com/DMPRoadmap/roadmap/pull/2735/commits/11d95923b05f988a42d12f8b5064168d9cc682ce

**Comment by briri** [comment]
sorry about that! GitHub has a 'Update branch' button that shows up when the branch being PRed isn't up to date. Its a bit dangerous I guess since it updates a repo I am not a member of!

**Comment by xsrust** [comment]
I'm additionally updating this branch, see https://github.com/DMPRoadmap/roadmap/pull/2741#issuecomment-732804820 for an explanation

**Review by briri — COMMENTED** [review]
looks good. thanks

**Review by briri — APPROVED** [review]

**Comment by pherterich** [comment]
Assuming that this is about dragging template sections around, that is working for me.

**Pull request merged**